cmd/policy_apply.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/pkg/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyApplyCmd() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "apply <orgName>/<policyPackName> <version>",
```

Comment:
nit/bikeshedding: why `apply` instead of `enable`? is there any way to disable a policy pack?

Reply:
Same reason as above—this is supposed to be a web UI-only feature. We will rip this out when it lands. Also, there is no "unapply" API call yet. :)